graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/kotlin2/Kotlin2TypeLookup.kt

```diff
@@ -171,9 +156,13 @@ class Kotlin2TypeLookup(
     /**
      * Returns the list of interfaces that this type implements
      */
-    fun implementedInterfaces(typeDefinition: ImplementingTypeDefinition<*>): List<String> {
-        return typeDefinition.implements
+    fun implementedInterfaces(typeDefinition: ImplementingTypeDefinition<*>, definitions: Collection<Definition<*>>): List<String> {
```

Comment:
The `document` is available above as a constructor param to this class - there should be no need for a new param here

Reply:
Fixed it, good catch